### ISSUE TITLE
Possible 64 vs. 32 bit mixup

### ISSUE THREAD

**Issue by weibel** [Visput/VPAttributedFormat#3]
The setValueByArgumentsValue consistently uses only the first argument in the arg list on 64 bit, at least when using the VPIdValueWrapper

```
#ifdef __LP64__
        // 64-bit processor architecture.
        // In 64-bit applications 'va_list' parameters are passed by reference.
        [conversionArgument.valueWrapper setValueByArgumentsPointer:&arguments];
#else
        // 32-bit processor architecture.
        // In 32-bit applications 'va_list' parameters are passed by value.
        [conversionArgument.valueWrapper setValueByArgumentsPointer:&arguments];
#endif
```
If setValueByArgumentsValue is substituted by the 32 bit setValueByArgumentsPointer it seems to work.

Are the 64 and 32 bit sections switched?

I'm testing on an iPhone 6.

**Comment by Visput** [comment]
Hm, It works on simulator, but doesn't work on real device.
Thanks for comment. Will see how to fix it.

**Comment by Visput** [comment]
Fixed in v1.2.1.
